incubator/airflow/values.yaml

```diff
@@ -0,0 +1,249 @@
+    ## header like the requested URL (no special configuration neeed)
+    annotations:
+      ##
+      ## Example for Traefik:
+      # traefik.frontend.rule.type: PathPrefix
+      # kubernetes.io/ingress.class: traefik
+  ##
+  ## Configure the flower endpoind
+  flower:
+    ##
+    ## if flower is '/airflow/flower':
+    ##  - Flower UI is at 'http://mycompany.com/airflow/flower'
+    ## NOTE: you need to have a reverse proxy/load balancer able to do URL rewrite in order to have
+    ## flower mounted on other path than root. Flower only does half the job in url prefixing: it
+    ## only generates the right URL/relative paths in the **returned HTML files**, but expects the
+    ## request to have been be at the root.
+    ## That's why we need a reverse proxy/load balancer that is able to strip the path
+    ## NOTE: do NOT keep trailing slash. For root configuration, set and empty string
+    path: /
+    ##
+    ## hostname for flower
+    host: ""
+    ##
+    ## Annotation for the Flower endpoint  ##
+    ## Please note their is a small difference between the way Airflow Web server and Flower handles
+    ## URL prefixes in HTTP requests:
+    ## Flower wants HTTP header to behave like there was no URL prefix, and but still generates
+    ## the right URL in html pages thanks to its `--url-prefix` parameter
+    ##
+    ##    Extracted from the Flower documentation:
+    ##    (https://github.com/mher/flower/blob/master/docs/config.rst#url_prefix)
+    ##
+    ##        To access Flower on http://example.com/flower run it with:
+    ##            flower --url-prefix=/flower
+    ##
+    ##        Use the following nginx configuration:
+    ##            server {
+    ##              listen 80;
+    ##              server_name example.com;
+    ##
+    ##              location /flower/ {
+    ##                rewrite ^/flower/(.*)$ /$1 break;
+    ##                proxy_pass http://example.com:5555;
+    ##                proxy_set_header Host $host;
+    ##              }
+    ##            }
+    annotations:
+      ##
+      ## NOTE: it is important here to have your reverse proxy strip the path/rewrite the URL
+      ## Example for Traefik:
+      # traefik.frontend.rule.type: PathPrefixStrip
+      # kubernetes.io/ingress.class: traefik
+
+##
+## Storage configuration
+persistence:
+  ##
+  ## enable persistance storage
+  enabled: false
+  ##
+  ## Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  # storageClass: default
+  accessMode: ReadWriteOnce
+  ##
+  ## Persistant storage size request
+  size: 1Gi
+
+##
+## Configure DAGs deployment and update
+dags:
```

Comment:
are we using this somewhere ? or planned for future ?

Reply:
I did it on kube-airflow but it is no more usable with the switch to puckle image. And I still have to update to the latest version of his image

Reply:
git-sync removed.

init-container works fine :)